testing/test_deploy_app.py

```diff
@@ -138,11 +138,18 @@ def insert_ssl_cert(args):
     util_run(("gsutil cp gs://%s/%s/* %s" % (SSL_BUCKET, args.mode, SSL_DIR)).split(' '))
   except Exception:
     logging.warning("ssl cert for %s doesn't exist in gcs" % args.mode)
-    return
-  util_run(("gcloud container clusters get-credentials %s --zone %s --project %s" %
-            (args.deployment, args.zone, args.project)).split(' '))
-  util_run(("kubectl create -f %s" % SSL_DIR).split(' '))
-
+    return True
```

Comment:
Should we use the retrying module rather than writing our own loop for retries?
The retrying module allows you to specify various conditions. Typically, I suggest retrying for up to some maximum amount of time with exponential backoff between retries.

Typically its best to specify a max duration for retries e.g. we will give up after X minutes; because that makes it easier to reason about.

Reply:
updated